x11-libs/goffice/goffice-0.10.43.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+GNOME2_LA_PUNT="yes"
+
+inherit gnome2
+
+DESCRIPTION="A library of document-centric objects and utilities"
+HOMEPAGE="https://git.gnome.org/browse/goffice/"
+SRC_URI="https://ftp.gnome.org/pub/GNOME/sources/goffice/0.10/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0.10"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~x64-solaris"
+IUSE="+introspection"
+
+# FIXME: add lasem to tree
+RDEPEND="
+	>=app-text/libspectre-0.2.6:=
+	>=dev-libs/glib-2.38.0:2
+	>=dev-libs/libxml2-2.4.12:2
+	dev-libs/libxslt
+	>=gnome-base/librsvg-2.22:2
+	>=gnome-extra/libgsf-1.14.24:=[introspection?]
+	>=x11-libs/cairo-1.10:=[svg]
+	>=x11-libs/gdk-pixbuf-2.22:2
+	>=x11-libs/gtk+-3.20:3
+	>=x11-libs/pango-1.24:=
+	x11-libs/libXext:=
+	x11-libs/libXrender:=
+	introspection? (
+		>=dev-libs/gobject-introspection-1:=
+		>=gnome-extra/libgsf-1.14.23:= )
+"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+	>=dev-util/gtk-doc-am-1.12
```

Comment:
Not sure if this was added before or after your branch but DEPEND is missing glib-utils.